css/style.css

```diff
@@ -1,18 +1,14548 @@
+.mdi-fw, .mdi-li {
```

Comment:
I guess the file was minified for performance, if there are no changes made to this file, it would be logical to change it back.

Reply:
okay